mne/epochs.py

```diff
@@ -3268,7 +3304,7 @@ def add_channels_epochs(epochs_list, verbose=None):
         raise ValueError('All epochs must be preloaded.')
 
     info = _merge_info([epochs.info for epochs in epochs_list])
-    data = [epochs.get_data() for epochs in epochs_list]
+    data = [epochs._data for epochs in epochs_list]
```

Comment:
I made this change because this test would fail otherwise: https://github.com/mne-tools/mne-python/blob/0156284d20f0b1ae2ca8630acf3a507b4b50b21d/mne/tests/test_epochs.py#L2488-L2490

that test is popping something from `self.info['chs']`, **not** updating `self._data`, to later trigger a RuntimeError in the test. The info and _data are thus "out of sync" (something that's not going to happen in normal use) and trip up the `get_data`.

Is my fix of using `._data` here safe? Do you have better ideas?

Reply:
> Is my fix of using ._data here safe? Do you have better ideas?

Probably good enough. The test is messing with stuff in non-standard ways (and using private functions) so it's okay to have to resort to something like `._data` if you couldn't find a cleaner way to do it after thinking about it.